internal/controller/nonadminbackup_controller.go

```diff
@@ -143,10 +211,28 @@ func (r *NonAdminBackupReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		logger.Error(err, "Failed to create backup", nameField, veleroBackupName)
 		return ctrl.Result{}, err
 	}
+	logger.Info("VeleroBackup successfully created", nameField, veleroBackupName)
+
+	// Phase: Created
```

Comment:
Similar comment as this one: https://github.com/migtools/oadp-non-admin/pull/13/files#r1576897507
IMHO we adopting a [ReconcileBatch](https://github.com/openshift/oadp-operator/blob/f23e821147c423c5698cde39784f72ac30cee3af/controllers/dpa_controller.go#L93) pattern as OADP Operator will be pretty helpful here. YMMV though.